block-producer/src/store/mod.rs

```diff
@@ -153,6 +164,12 @@ impl Store for DefaultStore {
             nullifiers.into_iter().collect()
         };
 
+        info!(
+            target: target!(),
+            account_hash = %format_opt(account_hash.as_ref()),
+            ?nullifiers,
+        );
```

Comment:
I would log these as a message rather than fields. But also, not sure we need to log it here (at least not at `INFO` level).